hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/util/TokenHandlerHelper.java

```diff
@@ -71,6 +71,8 @@
  * multiple handlers.
  */
 public class TokenHandlerHelper {
+    private static final int STAKING_REWARD_ACCOUNT = 800;
```

Comment:
I believe this must be read from the configuration.

Reply:
Ah yes agreed. Will change